config/locales/de.helpers.yml

```diff
@@ -18,6 +18,9 @@ de:
       new: "Neu"
       edit: "Bearbeiten"
       show: "Anzeigen"
+      assign_ownership: "Als Captain hinzufügen"
```

Comment:
Is the German word “Captain” for team owners common sense? Maybe also a question towards our PO @svitlanakalancha?

Reply:
I am pretty certain it isn't -- 'Besitzer', i thought, would've made it sound worse, though lol
let's talk about this tmrw